app/src/renderer/components/common/CookieBar.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div v-if="!session.cookiesAccepted" class="cookieBar">
+    <a class="close">
+      <i class="material-icons" @click="accept">close</i>
+    </a>
+    Lunie uses cookies and remote error collection to improve the app.
+    By dismissing this notification, you consent to this policy.
+    <a href="/privacy-policy">Click to learn more.</a>
```

Comment:
use vue-router

Reply:
I will add that when we have the actual policy